templates/Jenkinsfile

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/groovy
+
```

Comment:
Can we add some comment about this being an example file ... potentially adding all these files under an `example` directory to better indicate that it's just for example purposes?